### ISSUE TITLE
slightly more lines of stderr at opam failure

### ISSUE THREAD

**Issue by camlspotter** [ocaml/opam-repository#5860]
Currently opam shows the last 10 lines of stderr when it fails to install a package, but for packages which use OMake, it is too short to see the build error message.  For example:

```
### stderr ###
# [...]
# *** omake: targets were not rebuilt because of errors:
#    spotinstall.cmi
#       depends on: spotinstall.ml
#    spotinstall.cmo
#       depends on: spotinstall.ml
#    spotinstall.cmx
#       depends on: spotinstall.ml
#    spotinstall.o
#       depends on: spotinstall.ml
# E: Failure("Command 'sh -c \"cp OMakeroot.in OMakeroot; omake PREFIX=/home/travis/.opam/system\"' terminated with error code 2")
```

If opam prints 5 more lines, I could see the real cause:

```
Warning 44: this open statement shadows the value identifier sprintf (which is later used)
File "spotinstall.ml", line 1, characters 0-17:
Warning 44: this open statement shadows the value identifier & (which is later used)
File "spotinstall.ml", line 33, characters 17-30:
Error: Unbound value shell_command
```

Is it possible to tweak opam setting to print longer stderr, since it seems there is no way to access the full stderr file in Travis.

**Comment by rgrinberg** [comment]
Seems like this is a problem with opam itself. But I'd love to see this fixed as well since I also use omake.

**Comment by camlspotter** [comment]
Reported to `opam`: https://github.com/ocaml/opam/issues/2457

**Comment by camlspotter** [comment]
OPAMERRLOGLEN env var will change the stderr output length, but it will be available in OPAM 1.3.0.

**Comment by samoht** [comment]
Fixed upstream.
